README.md

```diff
@@ -4,38 +4,15 @@
 
 ## TiDB 文档结构
 
-目前，TiDB 的文档维护以下几个版本，分别放置在相应的 branch：
-
-- `master`：最新开发版
-- `release-4.0`：4.0 RC 版
-- `release-3.1`：3.1 稳定版
-- `release-3.0`：3.0 稳定版
-- `release-2.1`：2.1 稳定版
-
-TiDB 的文档结构如下：
```

Comment:
Because most of the files are put in the outermost layer, I don't see why this is important for a new contributor now, so I suggest we delete it. What do you think? @lilin90

Reply:
Agree.